src/main/java/dev/slimevr/gui/VRServerGUI.java

```diff
@@ -213,6 +216,50 @@ public void mouseClicked(MouseEvent e) {
 						});
 					}
 				});
+				add(Box.createHorizontalStrut(10));
+				add(floorClipButton = new JButton("Toggle Floor Clip") {
+					{
+						addMouseListener(new MouseInputAdapter() {
+							@Override
+							public void mouseClicked(MouseEvent e) {
+								boolean[] state = server.humanPoseProcessor.getLegTweaksState();
+								setFloorClipEnabled(!state[0]);
+							}
+						});
+					}
+				});
+
+				setFloorClipEnabled(
+					server
+						.getConfigManager()
+						.getVrConfig()
+						.getSkeleton()
+						.getToggles()
+						.get(SkeletonConfigToggles.FLOOR_CLIP.configKey)
+				);
+
+
```

Comment:
```suggestion
```

how come that there's two new lines here now?

Reply:
I did something weird on accident and ended up with a bunch of those must have missed that one.